homeassistant/components/update/const.py

```diff
@@ -0,0 +1,24 @@
+"""Constants for the update component."""
+from __future__ import annotations
+
+from typing import Final
+
+DOMAIN: Final = "update"
+
+# Bitfield of features supported by the update entity
+SUPPORT_SPECIFIC_VERSION: Final = 1
+SUPPORT_PROGRESS: Final = 2
+SUPPORT_BACKUP: Final = 4
```

Comment:
Should being able to update or not (at all) be a flag?
Right now this integration assumes updating is possible, which means that it cannot be used as just "binary sensor" for updates.

Reply:
Added `SUPPORT_INSTALL` in [`e4b005c` (#68248)](https://github.com/home-assistant/core/pull/68248/commits/e4b005cbddca036d6b1c0e8212f8f03495bc5371)